kornia/augmentation/container/video.py

```diff
@@ -187,8 +209,13 @@ def forward(  # type: ignore
             params = self.forward_parameters(input.shape)
 
         # Size of T
-        frame_num = input.size(self._temporal_channel)
-        input, label = self._input_shape_convert_in(input, label, frame_num)
+        if self.apply_inverse_func == InputApplyInverse or self.apply_inverse_func == MaskApplyInverse:
+            frame_num: int = input.size(self._temporal_channel)
+            input, label = self._input_shape_convert_in(input, label, frame_num)
+        else:
+            assert label is None
```

Comment:
```suggestion
            if label is not None:
                raise ValueError(f"Invalid label value. Got {label}")
```
`Issue: [B101:assert_used]`